app/api/v2/management/api_keys.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module API::V2
+  module Management
+    class APIKeys < Grape::API
```

Comment:
if this is specific to service account we should probably prefix the endpoint with "sa" or something

Reply:
or we rename uid to sa_uid so we could use this endpoint for managing either user api_keys or service account.